app/src/main/java/com/example/music_app/data/database/playlists/dao/PlaylistDao.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.music_app.data.database.playlists.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Upsert
+import com.example.music_app.data.database.playlists.models.PlaylistEntity
+import java.time.Instant
+
+@Dao
+interface PlaylistDao {
+    @Upsert
+    fun upsertPlaylist(playlistEntity: PlaylistEntity)
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
I don't think that `IGNORE` strategy would be the best choice here. Perhaps `REPLACE` would suit better